thumbnail/thumbnail_magick.go

```diff
@@ -368,23 +369,31 @@ func MakeThumbnailMagick(src io.Reader, dst http.ResponseWriter, params Thumbnai
 	defer mw.Destroy()
 	mw.SetResourceLimit(imagick.RESOURCE_THREAD, 1)
 
-	//画像入力
-	bytes, err := ioutil.ReadAll(src)
+	buf := make([]byte, 20)
```

Comment:
How do you decide the number 20 here?

Reply:
we need 12 bytes to decide the image info, so the 20 is enough. Any suggestion?

Reply:
How about just using 12 here? Or is it that most image types can be determined by 20 bytes, so we don't need to update this value when we add another file type? Either way, a small comment on the number may be good.